parsec/core/fs/workspacefs/sync_transactions.py

```diff
@@ -415,8 +415,18 @@ async def file_conflict(
                 await self.local_storage.set_manifest(parent_id, new_parent_manifest)
                 await self.local_storage.set_manifest(entry_id, other_manifest)
 
-                self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=new_manifest.id)
-                self._send_event(CoreEvent.FS_ENTRY_UPDATED, id=parent_id)
+                self._send_event(
+                    CoreEvent.FS_ENTRY_UPDATED,
+                    id=parent_id,
```

Comment:
Is there a reason why you changed the order of the events ?

Reply:
It was just to have the same order than in workspace/files creations, the ENTRY event in second.

Reply:
👍